### PR TITLE
Update exceljs to the latest version

### PR DESCRIPTION
**Pull request by San4es**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: QUnit IE
 
-on: [push]
+on: [push, pull_request]
```
Comment (IgnatovDan) [inline .github/workflows/CI.yml:3]: Change doesn't look to be related to exceljs version change.
Do we need it?

Comment (San4es) [reply .github/workflows/CI.yml:3]: No, but we saw problems with the new ExcelJS in IE. So I closed this PR.
